libnavui-androidauto/api/current.txt

```diff
@@ -283,7 +283,7 @@ package com.mapbox.androidauto.map.logo {
 package com.mapbox.androidauto.navigation {
 
   public final class CarActiveGuidanceMarkers implements com.mapbox.maps.extension.androidauto.MapboxCarMapObserver {
-    ctor public CarActiveGuidanceMarkers();
+    ctor public CarActiveGuidanceMarkers(com.mapbox.androidauto.routes.CarRoutesProvider carRoutesProvider = com.mapbox.androidauto.routes.NavigationCarRoutesProvider());
```

Comment:
I thought we would be able to delete the `NavigationCarRoutesProvider` because `MapboxNavigation` will provide the routes

Reply:
I don't see how we can do that with the current route preview implementation in Nav SDK. But I can make the constructor with a parameter `internal` and only keep the parameter-less constructor `public`, what do you think?

Reply:
Yeah you're right, we also have the same thing to resolve with `CarRouteLineRenderer` and `CarNavigationCamera`

We are not entirely migrated yet to be able to remove this